src/app/modules/main/chat_section/chat_content/input_area/gif_column.nim

```diff
@@ -0,0 +1,54 @@
+import NimQml, Tables, sequtils
+
+import ../../../../../../app_service/service/gif/dto
+
+type
+  GifRoles {.pure.} = enum
+    Url = UserRole + 1
+    Id = UserRole + 2
+    Title = UserRole + 3
+    TinyUrl = UserRole + 4
+
+QtObject:
+  type
+    GifColumn* = ref object of QAbstractListModel
```

Comment:
not sure but can this be renamed to something which has model in its name? easier to follow. also that how most other QAbstractListModel have been declared so far

Reply:
I can add model

Reply:
thank you !